### ISSUE TITLE
Crash when reordering items

### ISSUE THREAD

**Issue by beaucollins** [mattrubin/Authenticator#135]
Steps to reproduce:

1) Add enough tokens to require scrolling.
2) Scroll to the bottom of the table
3) Tap "Edit"
4) Drag the last token upwards so it is no longer the last item
5) Crash

```
2016-10-10 09:40:52.195995 Authenticator[3852:1015298] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Attempt to create two animations for cell'
```

This happens on `master`.

**Comment by beaucollins** [comment]
If I remove [this line](https://github.com/mattrubin/Authenticator/blob/28d32f129f34ee82912d5d6290cc4249a06384e6/Authenticator/Source/TokenListViewController.swift#L208) the crash no longer happens.

I don't think it's necessary to call `reloadRowsAtIndexPaths` because it will only happen to cells that are off screen given that it's in the `else` branch of `cellForRowAtIndexPath` so they will be updated with the correct view state when they are scrolled back into view.

**Comment by mattrubin** [comment]
Fixed by #138.
